test/unit/nedap/utils/test/api.cljc

```diff
@@ -190,7 +201,24 @@
           `{:type :fail, :expected (~'= 0 1), :actual (~'not (~'= 0 1))}
 
           (sut/expect (swap! a inc) :with = :to-change @a :from 1 :to 3)
-          `{:type :fail, :expected (~'= (deref ~'a) 3), :actual ~'(not (= 2 3))}))))
+          `{:type :fail, :expected (~'= (deref ~'a) 3), :actual ~'(not (= 2 3))}
+
+          ;; change matcher to `match?`
+          (sut/expect 0 :with match? :to-change 0 :from 0 :to 1)
+          `{:type :fail,
+            :expected (~'match? 0 1),
+            :actual ~'{:summary (not (matcher-combinators.clj-test/match? 0 1)),
```

Comment:
I hope these translate nicely into a correct developer and CI experience when a given test fails.

Seems a good QA plan